tests/PySys/mqtt_port/mqtt_port_unset/run.py

```diff
@@ -0,0 +1,55 @@
+import sys
+import time
+
+from pysys.basetest import BaseTest
+
+
+"""
+Validate changing the mqtt port using the tedge command
+
+Given a configured system
+When mqtt.port is set using tedge mqtt.port set with sudo
+When the tedge config is listed and searched for the port that has been set 
+
+"""
+
+
+class MqttPortUnSet(BaseTest):
+    def setup(self):
+        self.tedge = "/usr/bin/tedge"
+        self.sudo = "/usr/bin/sudo"
+
+        # set a new mqtt port for local communication
+        mqtt_port = self.startProcess(
+            command=self.sudo,
+            arguments=[self.tedge, "config", "set", "mqtt.port", "8880"],
+            stdouterr="mqtt_port_set",
+        )
+
+        self.addCleanupFunction(self.mqtt_cleanup)
```

Comment:
We miss validation at this point that the configuration has been change and is not a default value, meaning that you when you call `execute` are not in an unknown state.

@abelikt is there PySys way to do it?

Reply:
Hmm, not sure if there is a good PySys way for that. But we could add a TedgeBaseTest Environment with some helper functions that help to validate config file content.